src/catalog/src/system_schema/information_schema/tables.rs

```diff
@@ -234,17 +237,45 @@ impl InformationSchemaTablesBuilder {
             .context(UpgradeWeakCatalogManagerRefSnafu)?;
         let predicates = Predicates::from_scan_request(&request);
 
+        let information_extension = utils::information_extension(&self.catalog_manager)?;
+
+        // TODO(dennis): `region_stats` API is not stable in distributed cluster because of network issue etc.
+        // But we don't want the statments such as `show tables` fails.
```

Comment:
@WenyXu  Looks like the `region_stats` API will return an error in instance test 

https://github.com/GreptimeTeam/greptimedb/blob/7a4276c24a1396bc7a8e6df998799dbecdb838d0/tests-integration/src/tests/instance_test.rs#L1478

The failure CI is https://github.com/GreptimeTeam/greptimedb/actions/runs/11657280986/job/32454614252?pr=4927

```
thread 'tests::instance_test::test_use_database::case_2_test_with_distributed' panicked at /Users/dennis/programming/rust/greptimedb/src/common/test-util/src/recordbatch.rs:26:75:
called `Result::unwrap()` on an `Err` value: 0: , at src/common/recordbatch/src/adapter.rs:291:55
1: External(External(0: , at src/common/recordbatch/src/adapter.rs:291:55
1: External(0: Failed to region stats in cluster, at src/cmd/src/lib.rs:172:14
```

So I have to use `unwrap_or_else` instead of `?` operator here.

Reply:
We can call `enable_access_cluster_info` in `tests-integration/cluster`'s `build_frontend` method
```rust
    async fn build_frontend(
        &self,
        metasrv: MockInfo,
        datanode_clients: Arc<NodeClients>,
    ) -> Arc<FeInstance> {
        let mut meta_client = MetaClientBuilder::frontend_default_options(1000)
            .channel_manager(metasrv.channel_manager)
            .enable_access_cluster_info()
            .build();
```

Reply:
Created an issue to track it #4935 , can you comment into it? I think it's a good first issue.